cmake/std/atdm/README.md

```diff
@@ -214,7 +214,9 @@ is not `CLANG`, then these options are ignored.)
 **`<fpic>`:** The following `<build-name>` keyword will result in `-fPIC`
 being added to `CMAKE_CXX_FLAGS`:
 
-* `fpic`: Add `-fPIC` to `CMAKE_CXX_FLAGS`
+* `fpic`: Turn on `CMAKE_POSITION_INDEPENDENT_CODE`
```

Comment:
Can we mention that `fpic` is the default option on all platforms?  Otherwise, the documentation is not clear here.